### PR TITLE
Group all migration code as it is still in dev

### LINKED ISSUE #141: Merge upgrade code to 2.0.0 and dev versions into dev
To cleanup current upgrade process.

review by the way the upgrade code which choosed the upgrade scripts to run. Dev must be the last executed.

### PR DESCRIPTION
**Pull request by btry**
Fix #141

### CODE REVIEW

**Comment by ajsb85** [comment]
Hi, @DIOHz0r Could you review this PR?

Thanks.

**Comment by btry** [comment]
@ajsb85 not needed he alerady got a mail notif

File: install/upgrade/update_to_dev.php [hunk 1]
```diff
@@ -119,15 +158,16 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $table = 'glpi_plugin_flyvemdm_packages';
    $migration->addField($table, 'parse_status', "enum('pending', 'parsed', 'failed')",
       ['after' => 'dl_filename', 'default' => 'pending']);
+   $migration->addfield($table, 'plugin_orion_tasks_id', 'integer', ['after' => 'dl_filename']);
```
Comment (DIOHz0r) [inline install/upgrade/update_to_dev.php]: Will it be necessary to add a key for this field?

Comment (btry) [reply install/upgrade/update_to_dev.php]: update_to_2_0.php is deleted, its conteint is simply moved in update_to_dev.php, and I grouped the upgrade code per table. Many calls to addKey() were mostly duplicated between each file.

**Pull request merged**